docs/specification.md

```diff
@@ -766,6 +783,8 @@ Note that operator relation is not transitive.
 
 One more uncertainty is related to the `token_metadata` entrypoint: whether metadata is constant or can be changed during contract's lifetime.
 We assume that token metadata stays constant forever.
+Upd: We have an additional entrypoint that holds all token metadata
```

Comment:
>Upd:

Not sure if this is a left over.

Reply:
No, it's an commentary that  needs to be discussed